src/sharedwithyou.cs

```diff
@@ -188,23 +188,41 @@ interface SWHighlightCenter
 		[Export ("getHighlightForURL:completionHandler:")]
 		void GetHighlight (NSUrl urL, Action<SWHighlight, NSError> completionHandler);
 
+		[NoTV]
 		[Export ("collaborationHighlightForIdentifier:error:")]
 		[return: NullAllowed]
 		SWCollaborationHighlight GetCollaborationHighlight (string collaborationIdentifier, [NullAllowed] out NSError error);
 
+		[NoTV]
 		[Async]
 		[Export ("getCollaborationHighlightForURL:completionHandler:")]
 		void GetCollaborationHighlight (NSUrl url, Action<SWCollaborationHighlight, NSError> completionHandler);
 
+		[NoTV]
 		[Export ("postNoticeForHighlightEvent:")]
 		void PostNotice (ISWHighlightEvent @event);
 
+		[iOS (16,1), NoTV]
+		[Export ("clearNoticesForHighlight:")]
+		void ClearNotices (SWCollaborationHighlight highlight);
+
+		[NoTV]
 		[Async]
 		[Export ("getSignedIdentityProofForCollaborationHighlight:usingData:completionHandler:")]
 		void GetSignedIdentityProof (SWCollaborationHighlight collaborationHighlight, NSData data, Action<SWSignedPersonIdentityProof, NSError> completionHandler);
 	}
 
-
+#if __MACOS__
+	[NoWatch, Mac (13,0), NoTV, NoiOS, NoMacCatalyst]
```

Comment:
Q: If this is in a `__MACOS__` test, do you need `NoWatch`, `NoTV`, `NoiOS` and `NoMacCatalyst`?

Reply:
Ideally we wouldn't have any #if ... inside api definitions, so if it could be removed, that would be much better.